librz/include/rz_util/rz_str.h

```diff
@@ -274,6 +276,8 @@ typedef struct rz_str_stringify_opt_t {
 RZ_API RzStrEnc rz_str_guess_encoding_from_buffer(RZ_NONNULL const ut8 *buffer, ut32 length);
 RZ_API RZ_OWN char *rz_str_stringify_raw_buffer(RzStrStringifyOpt *option, RZ_NULLABLE RZ_OUT ut32 *length);
 
+RZ_API const char *rz_str_indent(int indent);
```

Comment:
is this really necessary as an API? Is it used in multiple places or just in one place? Because I'm tempted to say this should be a static function somewhere.

Reply:
Lets keep it here. Someone asked @imbillow to move it to the RzUtil, moving back and forth isn't exactly encouraging.